src/schema/ast.rs

```diff
@@ -451,34 +466,19 @@ impl DirectiveLocation {
             EnumValue => "ENUM_VALUE",
             InputObject => "INPUT_OBJECT",
             InputFieldDefinition => "INPUT_FIELD_DEFINITION",
+            VariableDefinition => "VARIABLE_DEFINITION",
         }
     }
 
     /// Returns `true` if this location is for queries (execution)
     pub fn is_query(&self) -> bool {
         use self::DirectiveLocation::*;
         match *self {
-            Query
-            | Mutation
-            | Subscription
-            | Field
-            | FragmentDefinition
-            | FragmentSpread
-            | InlineFragment
-                => true,
-
-            Schema
-            | Scalar
-            | Object
-            | FieldDefinition
-            | ArgumentDefinition
-            | Interface
-            | Union
-            | Enum
-            | EnumValue
-            | InputObject
-            | InputFieldDefinition
-                => false,
+            Query | Mutation | Subscription | Field | FragmentDefinition | FragmentSpread
+            | InlineFragment => true,
+
+            Schema | Scalar | Object | FieldDefinition | ArgumentDefinition | Interface | Union
+            | Enum | EnumValue | InputObject | InputFieldDefinition | VariableDefinition => false,
```

Comment:
This made code worse, and hard to compare. The original formatting is not an accident it's done to have it easier to compare.

Reply:
You can use `#[rustfmt::skip]` here FWIW